build.cmd

```diff
@@ -188,8 +188,8 @@ dir /b "%__DotNetCliPath%\sdk"
 call "!VS%__VSProductVersion%COMNTOOLS!\VsDevCmd.bat"
 echo Commencing build of managed components for %__BuildOS%.%__BuildArch%.%__BuildType%
 echo.
-set "__BuildLog=%__LogsDir%\msbuild_%__BuildOS%__%__BuildArch%__%__BuildType%.log"
-%_msbuildexe% /ConsoleLoggerParameters:ForceNoAlign "%__ProjectDir%\build.proj" %__MSBCleanBuildArgs% /p:RepoPath="%__ProjectDir%" /p:RepoLocalBuild="true" /p:RelativeProductBinDir="%__RelativeProductBinDir%" /p:NuPkgRid=win7-x64 /p:ToolchainMilestone=%__ToolchainMilestone% /nologo /maxcpucount /verbosity:minimal /nodeReuse:false /fileloggerparameters:Verbosity=normal;LogFile="%__BuildLog%"
+set "__BuildLog=%__LogsDir%\Managed_%__BuildOS%__%__BuildArch%__%__BuildType%"
+%_msbuildexe% /ConsoleLoggerParameters:ForceNoAlign "%__ProjectDir%\build.proj" %__MSBCleanBuildArgs% /p:RepoPath="%__ProjectDir%" /p:RepoLocalBuild="true" /p:RelativeProductBinDir="%__RelativeProductBinDir%" /p:NuPkgRid=win7-x64 /p:ToolchainMilestone=%__ToolchainMilestone% /nologo /maxcpucount /verbosity:minimal /nodeReuse:false /flp:Summary;Verbosity=minimal;LogFile=%__BuildLog%.sum /flp1:warningsonly;logfile=%__BuildLog%.wrn /flp2:errorsonly;logfile=%__BuildLog%.err /flp3:Verbosity=normal;LogFile=%__BuildLog%.log /flp4:Verbosity=diagnostic;LogFile=%__BuildLog%.diag
 IF NOT ERRORLEVEL 1 (
```

Comment:
I do not have numbers handy for current state of corert repo, but I have seen the diagnostic logging to be tracked down to be source of slow build times number of times before. See https://github.com/dotnet/coreclr/pull/1211 for example.

When you are hacking on a build, you can turn it and there should be an easy way to do it. But it should not be on by default.